unified_planning/test/test_conditional_effects_remover.py

```diff
@@ -64,7 +65,7 @@ def test_basic_conditional(self):
 
         with OneshotPlanner(problem_kind=unconditional_problem.kind()) as planner:
             self.assertNotEqual(planner, None)
-            uncond_plan = planner.solve(unconditional_problem).plan
+            uncond_plan = planner.solve(unconditional_problem, out=sys.stdout).plan
```

Comment:
Unless you use the output in the test, I would keep the tests silent

Reply:
Yes, True, this was a debug feature, forgotten to remove..